src/app/shared/page/header/header.component.html

```diff
@@ -51,13 +51,41 @@
                 routerLinkActive="font-bold hover:border-none"
                 class="transition-all duration-150 hover:border-b-4 hover:border-teal-500"
               >
-                {{ link.label }}
+                {{ link.label | translate }}
               </a>
             </li>
           </ul>
         </div>
 
         <div class="hidden lg:flex lg:gap-3">
+          <div class="btn-group" dropdown>
+            <button
+              dropdownToggle
+              type="button"
+              class="bg-slate-200 hover:bg-slate-300 w-10 h-10 rounded-lg ml-3"
+            >
+              <i class="fa-light fa-globe fa-xl text-teal-500"></i>
+            </button>
+            <ul *dropdownMenu class="dropdown-menu min-w-fit" role="menu">
+              <li role="menuitem">
+                <button
+                  class="dropdown-item dropdown-item hover:bg-teal-500 hover:text-white active:bg-teal-500"
+                  (click)="translate.use('ar')"
+                >
+                  AR
+                </button>
+              </li>
+              <li role="menuitem">
+                <button
+                  class="dropdown-item dropdown-item hover:bg-teal-500 hover:text-white active:bg-teal-500"
+                  (click)="translate.use('en')"
+                >
+                  EN
+                </button>
+              </li>
```

Comment:
- use loop
- no need to use element inside element use `li` or `button`
- remove duplicated class `dropdown-item`
- use method `changeLang(language)`